pkg/errors/errors.go

```diff
@@ -25,14 +25,18 @@ import (
 )
 
 const (
-	launchTemplateNotFoundCode = "InvalidLaunchTemplateName.NotFoundException"
+	launchTemplateNameNotFoundCode = "InvalidLaunchTemplateName.NotFoundException"
+
+	//The specified launch template ID does not exist
+	launchTemplateIDNotFoundCode = "InvalidLaunchTemplateId.NotFound"
```

Comment:
nit: Do we need to type this separately or can this just go into the `notFoundErrorCodes` bucket since we don't have a specific check that the error is a launchTemplateIDNotFound error?

Reply:
Can do that.